src/axom/primal/geometry/NURBSPatch.hpp

```diff
@@ -1269,6 +1463,14 @@ class NURBSPatch
     }
 
     std::swap(m_knotvec_u, m_knotvec_v);
```

Comment:
Please use [axom::utilities::swap](https://github.com/LLNL/axom/blob/44562f92a400204e33915f48b848eb68e80a1bf1/src/axom/core/utilities/Utilities.hpp#L105) as a tiny step towards future device support.

Reply:
Changed it here, and elsewhere.